File: src/Phoenix.Models/Plcs/Rvds/Dto/Rvd145ChartDto.cs
using Phoenix.Models.Base.Dto;

namespace Phoenix.Models.Plcs.Rvds.Dto
{
   public sealed class Rvd145ChartDto : PlcChartDtoBase
   {
      public float OutsideTemp { get; init; }
      public float ChHighInletPresure { get; init; }

      public float Ch1HighOutletTemp { get; init; }
      public float Ch1LowInletTemp { get; init; }
      public float Ch1LowOutletPresure { get; init; }
      public float Ch1HeatCurveTemp { get; init; }

      public float DhwTemp { get; init; }
      public float DhwTempSet { get; init; }
   }
}

File: src/Phoenix.Web/src/models/api/plcs/rvds/dto/rvd145ChartDto.ts
import { PlcChartDtoBase } from '../../../base/dto/plcChartDtoBase';

export interface Rvd145ChartDto extends PlcChartDtoBase {
   outsideTemp: number;
   chHighInletPresure: number;
   ch1HighOutletTemp: number;
   ch1LowInletTemp: number;
   ch1LowOutletPresure: number;
   ch1HeatCurveTemp: number;
   dhwTemp: number;
   dhwTempSet: number;
}
